Assets/Scripts/MarchingCubes/Chunk.cs

```diff
@@ -0,0 +1,60 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+public class Chunk : MonoBehaviour
+{
+
+    int index;
+    int resolution;
+    MeshFilter meshFilter;
+    MeshCollider meshCollider;
+    Mesh mesh;
+    MarchingCubes marchingCubes;
+    PillPlayerController player;
+    MinMaxTerrainLevel terrainLevel;
+
+    /// <summary>
+    /// Initialize
+    /// </summary>
+    /// <param name="index"></param>
+    /// <param name="resolution"></param>
+    /// <param name="marchingCubes"></param>
+    /// <param name="player"></param>
```

Comment:
Empty and seemingly out od date param summary. Either leave out or document

Reply:
fixed